scripts/_helpers.py

```diff
@@ -150,3 +151,88 @@ def dlProgress(count, blockSize, totalSize):
 
     urllib.request.urlretrieve(url, file, reporthook=dlProgress)
 
+
+def mocksnakemake(rulename, **wildcards):
```

Comment:
```suggestion
def mock_snakemake(rulename, **wildcards):
```

Name suggestion. Calling it that way includes an imperative.

Reply:
yes, that's perhaps better, I'll do that in a seperate commit